### ISSUE TITLE
IGAPP-305: Prepare and translate override translations

### ISSUE THREAD

**Issue by sarahsporck** [sarahsporck/Migration-Test-Repository#55]
# steffenkleinle - 8.10.2020, 10:11:36

The Aschaffenburger probably want to have some locales with other values/translations than in the integreat app. Provide them with csvs/ods files for adjustments and translation.

Make sure the locales are actually translated afterwards.

·         Englisch

·         Arabisch

·         Rumänisch

·         Chinesisch <-- That's a new one!! 

·         Polnisch

·         Russisch

·         Spanisch

·         Französisch

·         Italienisch

·         Türkisch 

**Acceptance Criteria**
* New languages have to be added to the locales.json and the language key to the config.json file
* The overrides json file has to be added

**Environment: -**
**Linked issues:**
- Blocks: IGAPP-251
- Blocks: IGAPP-304
- Relates: NATIVE-552

**Comment by sarahsporck** [comment]
**Fritjof Knier - 24.2.2021, 16:38:54**

Translations have moved to progress today. We can expect to receive them on Friday or Monday.

**Comment by sarahsporck** [comment]
**Fritjof Knier - 27.2.2021, 08:55:15**

<<sub>max.ammann> und <</sub>steffen.kleinle> 
Translations have arrived. I only checked for completion and I have two remarks for you. 

Since Italian and Chinese were new languages both Translation included the "wohnen"-translations which I told tolingo not to translate but this happened: 
* In Italian, the translator inserted the German translation into the cell of the target language
* In Chinese they seem to have translated the Translations as there is content in the cells (which DeepL tells me is the right translations)

I did have a few more override translations due to the special wording of the Aschaffenburg-App. In some cases we would have had Expressions like '\`appName`-App' so I hopefully got all those cases correctly into the override file. 

Let me know if you run into any trouble here.
